package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "build:watch": "node ./builder/program -w",
+    "build:watch:theme": "node builder/program.js -t ./themes/_default.yaml -w",
```

Comment:
👍

Reply:
I didn't see the point of this one. This is what happens by default, If you look at the `builder.js` file.